addons/sitting/functions/fnc_getRandomAnimation.sqf

```diff
@@ -46,4 +46,4 @@ _animations = [
 ];
 
 // Select random animation
-(_animations select (floor (random (count _animations))))
+(_animations select floor random count _animations)
```

Comment:
It doesn't do anything for performance. It's just a personal preference to keep the amounts of required parenthesis to a minimum whenever I can, considering they often lead to a confusing looking syntax.